content/getting-started/javascript.textile

```diff
@@ -0,0 +1,131 @@
+
+```[javascript]
+const channel = realtimeClient.channels.get('my-first-channel');
+
+await channel.subscribe((message) => {
+  console.log(`Received message: ${message.data}`);
+});
+```
+
+* Use the Ably CLI to publish a message to your first channel: @ably channels publish my-first-channel 'Hello!'@. The message will be received by the client you've subscribed to the channel, and be logged to the console.
+
+* In a new terminal tab, subscribe to the same channel using the CLI: @ably channels subscribe my-first-channel@. Publish another message using the CLI and you will see that it's received instantly by the client you have running locally, as well as the subscribed terminal instance.
+
+h2(#step-3). Step 3: Join the presence set
+
+Presence enables clients to be aware of one another if they are present on the same channel. You can then show clients who else is online, provide a custom status update for each, and notify the channel when someone goes offline.
+
+* Add the following lines to your @getStarted()@ function to subscribe to, and join, the presence set of the channel. Then run it:
+
+```[javascript]
+await channel.presence.subscribe((member) => {
+  console.log(`Event type: ${member.action} from ${member.clientId} with the data ${JSON.stringify(member.data)}`)
+});
+
+await channel.presence.enter("I'm here!");
+```
+
+* In the "dev console":https://ably.com/accounts/any/apps/any/console of your first app, attach to @my-first-channel@. Enter a @clientId@, such as @my-dev-console@, and then join the presence set of the channel. You'll see that @my-first-client@ is already present in the channel. In the console of your browser or IDE you'll see that an event was received when the dev console client joined the channel.
+
+* You can have another client join the presence set using the Ably CLI: @ably channels presence enter my-first-channel --client-id "my-cli" --data '{"status":"online"}'@.
+
+h2(#step-4). Step 4: Retrieve message history
+
+You can retrieve previously sent messages using the history feature. Ably stores all messages for 2 minutes by default in the event a client experiences network connectivity issues. This can be extended for longer if required.
+
+If more than 2 minutes has passed since you published a regular message (excluding the presence events), then you can publish some more before trying out history. You can use the Pub/Sub SDK, Ably CLI or the dev console to do this.
+
+For example, using the Ably CLI to publish 5 messages: @ably channels publish --count 5 my-first-channel "Message number {{.Count}}"@
+
+* Add the following lines to your @getStarted()@ function to retrieve any messages that were recently published to the channel. Then run it:
+
+```[javascript]
+const history = await channel.history();
+  console.log(history.items.map((message) => message.data));
+```
+
+The output will look similar to the following:
+
+```[json]
+[
+  'Message number 1',
```

Comment:
History responses are returned in reverse chronological order by default (newest first), so this would actually be 5, 4, 3, 2, 1